plover/messages/plover.pot

```diff
@@ -1141,17 +1144,21 @@ msgstr ""
 msgid "no suggestions"
 msgstr ""
 
-#: plover/gui_qt/trayicon.py:115
+#: plover/gui_qt/suggestions_widget_ui.py:31
+msgid "Form"
+msgstr ""
```

Comment:
Where is that coming from? Can you run ensure the generated UI files are up-to-date (`setup.py build_ui -f`) before running `./setup.py extract_messages`.

Reply:
It's gone now. I had to manually remove `_ui.py` files first (`-f` does not seem to do this?)

Reply:
That's because it's a left-over, the UI file was removed in cc794c242dacd1e0cd13275aafe38bff7e910a8f.